### PR TITLE
Tweak: Replacing Main Page link to google groups with discord link

### PR DESCRIPTION
**Pull request by Zitchas**
**Bugfix:**

## Problem
Currently, the front page of the Endless Sky repository directs people who are interested in contributing to the google groups page. As far as I can tell, none of the core developers or content creators frequent that group. Nor does the majority of the community that regularly interacts with the developers or content creators. As such, directing people to the google groups is likely to give outsiders the strong impression that there is virtually no activity or discussion surrounding the game's development.

## Fix Details
This replaces the link to the google groups with a link to the Endless Sky Community discord. it also changes the link label so that it is clear that it is linking to the discord instead of to the google forum.

## Results

So, 6 out of 23 people with google accounts that have visited the google groups previously can't reach it, and 0 out of 78 attempts to visit it for a "first" time succeeded.

## Reasons to make this switch:
1. **Appearances:** Having the link to the developer discussion area be a link to somewhere that newcomers can't reach presents our development community as either: a) inactive/dead, or b) secretive/deceptive. Basically, if they know development is ongoing, then having a link to our developer discussion being an unreachable forum just makes us look elitist and secretive. For those who don't know that we're active, then we just look like another abandoned project.
2. **Habit:** With a dead link, people are likely to find our github, try the link, discover it is dead, and never try it again. This means that people who come back to the site later aren't likely to try it, because they've already tried it and it was dead. This means that right now, having a dead link prominently on our page decreases the chance of people going to the google groups once it is functional.
3. **Honesty:** If the google group is moved to an Archival section, it is clear to everyone that it is not currently being used, and that people shouldn't expect its accessibility (or lack thereof) to set any expectations about our community.
4. **New-Factor:** In the future, when/if the google groups becomes functional again, we can move the google groups link back to prominence somewhere as the restored google groups. This change will draw attention to the link as a visual indicator that it has been fixed and that people should use it again.
5. **MZ reminder:** If this link gets moved to an Archival section, it might serve as a reminder for MZ when he visits the page that this needs to be fixed. Thus it could contribute to it actually being fixed.
6. **Time:** This change is a very simple change. It costs mere seconds for someone to apply it right now, and less than a minute to do a new PR to change it once the google groups are fixed. If it helps, I can commit to making a new PR that implements this immediately after this PR is merged so that when the google groups is fixed, someone just has to spend the couple of seconds to apply that PR.

### CODE REVIEW

**Comment by MCOfficer** [comment]
Refs https://github.com/endless-sky/endless-sky.github.io/issues/23, https://github.com/endless-sky/endless-sky.github.io/pull/24

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Most development is done on Linux and Windows, using the [SCons](https://scons.o
 For full installation instructions, consult the [Build Instructions](https://github.com/endless-sky/endless-sky/wiki/BuildInstructions) wiki page. Some additional information may be found in the "readme-developer" text file.
 
 # Contributing
-As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, while there is a loosely defined [development roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to contribute are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [discussion forum](http://groups.google.com/group/endless-sky).
+As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, while there is a loosely defined [development roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to contribute are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [discussion discord](https://discord.gg/ZeuASSx).
```
Comment (MCOfficer) [inline README.md:28]:
```suggestion
As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, while there is a loosely defined [development roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to contribute are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [community discord](https://discord.gg/ZeuASSx).
```

**Review by MCOfficer — APPROVED** [review]

**Comment by Terin** [comment]
I had tossed that into my PR for updating the build instructions, but it makes sense to split it off. Although I [included a link to the Steam forums](https://github.com/endless-sky/endless-sky/pull/5704/commits/dd1e00e06e522f10daab4d936c3cf659817bf9ca), as well, in place of the Google forums. Should that be the case here, also? Or just the Discord link?

**Comment by Zitchas** [comment]
Hmmm, I'm undecided.

The steam forums, I feel, are basically just there for people trapped in the steam ecosystem. I don't see much value in trying to bring people from outside steam into it.

I must admit, part of me feels that it would be best to be complete in having all three listed together. But at the same time, I feel that linking people who want to contribute to the game anywhere but the discord (or GitHub itself) is doing them a diservice to the lack of developer or contributor knowledge on either of the other two ecosystems. Putting the three together implies that they are equal, and they are not.

**Comment by Terin** [comment]
As useful as the Github repo is, it's not exactly equivalent to forums. Derpy gives a decent amount of attention to them, and from my understanding, he'd like to re-arrange them some more if he had administrator access. They don't have complete developer attention, but enough to make them worth putting in the readme, I feel.

**Review by Pointedstick — APPROVED** [review]

**Comment by Zitchas** [comment]
The more time goes on, the more this change needs to happen. The google groups looks deader every time I go there. This time when following the link to it, I even got a brief error message. 

Two points:
1) The last activity on the Google Groups was a Feb 22 post by myself that was just a PSA pointing people to the discord. Prior to that, the last post was Jan 26. Given that level of activity, we need to either just delete the link to the google groups entirely, or replace it with a link to the discord. At this point, it would be better for everyone to *not* have a link to an official community at all than to direct people to a forum that is both dead and (at this point) filled with out-of-date information.
2) I don't know what changed, might have been part of the "Google groups has changed" message, but the groups doesn't seem to have sub-groups anymore. Or at least I can't find them. It just looks like one giant wall of threads now. That much less useful.

tldr: The link to the google groups needs to be removed. What we talk about what to do with its spot, but the link itself is an active **negative** for the community at this point.

**Comment by Amazinite** [comment]
If we're going to make any change, I don't think that solely pointing people toward the Discord server is in order. Perhaps the Discord server could be the first listed, but we should list the Google Group and probably also the Steam forums as other valid places to go looking for discussion.

**Comment by Zitchas** [comment]
Sure, adding in the steam forums as valid places to go is valid. I can look at adding it.

But the google groups is *dead.* That link is basically pointing to the forum equivalent of a corpse. It is worse than useless. It actively makes the game look like its community died.

**Comment by Zitchas** [comment]
Also, I'll point out that the sentence is specifically:
> Those who wish to contribute are encouraged to review the wiki, and to post in the discussion forum.

I don't know about anyone else, but if I want to contribute, being directed to post in a dead forum is a very fast way to discourage anyone from learning how to contribute.

**Comment by Zitchas** [comment]
OK @Amazinite, revised it a bit:

# Contributing
As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, while there is a loosely defined [development roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to contribute are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [community discord](https://discord.gg/ZeuASSx). Those who prefer to use Steam can use the [steam discussion room](https://steamcommunity.com/app/404410/discussions/).

# Archival
Endless Sky's main discussion and development area used to be [here on google groups](https://groups.google.com/g/endless-sky) (and an [alternative link](http://groups.google.com/group/endless-sky), but it is no longer used regularly and is included here solely for historical purposes. Note that it may require a google account to view.

**Comment by Zitchas** [comment]
For the sake of semi-scientific informed decision making, I'm running a poll on the discord:
https://discord.com/channels/251118043411775489/509076786538086400/843148387657842719

The results are early so far, but there does seem to be a strong trend so far:
6 out of 9 could access the group when they tried while logged into an account that has visited it before.
0 out of 10 could access it when they tried while logged into an account that had NOT visited it before.
0 out of 12 could access it when they were not logged into an account.

**Comment by GalacticRuler** [comment]
I had been in the group before while signed in, but could not get to it without looking at the specific page for google groups you've been in

google's search functions returned nothing when attempting to find it


cross posted here at Z's request

**Comment by Zitchas** [comment]
Google Groups Accessibility poll (after 19h)

Here are the results of the poll after about 19h.

**Comment by Zitchas** [comment]
In view of the points raised by @tehhowch in the [google groups thread,](https://groups.google.com/g/endless-sky/c/00HtWOibjOg)  I would like to suggest that this current  change be merged for now (because right now and for as far as we can see into the future) there is zero point in pointing people to the google groups.

If, at some point in the future, the google groups (or some other forum) is implemented, we can just as easily adjust this page to point to it. The important thing is that we don't drive new contributors away by continuing to point them to a non-accessible resource.

**Comment by Zitchas** [comment]
The final results of the poll:

For accounts that **have** visited it before: (note, for the sake of the poll I had to give the first one for each one, which I have removed to include my own test)
17 : I can view the group while signed into an account that has visited it before.
6 : I can NOT view the group while signed into an account that has visited it before.

For accounts that have NOT visited it before:
0 : I can view the group while signed into an account that has never visited it before.
48 : I can NOT view the group while signed into an account that has never visited it before.

While not signed into a google account:
0: I can view the group while not signed into any google account.
30: I can NOT view the group while not signed into any google account. 

Poll on Google Groups Accessibility 2021-June-04


So, 6 out of 23 people with google accounts that have visited the google groups previously can't reach it, and 0 out of 78 attempts to visit it for a "first" time failed.

I fully agree with Tehhowch that it would be great to get the google groups functional again. They are great for providing topic-continuity instead of having topics drop off the radar only to be revived again and again. However, all the utility in the world can't make up for the fact that it is difficult for people who have already been there to access it; and flat out impossible for anyone new to reach it. As such, listing on our github page that it is where new people should be going to figure out how to take part in development is a flat out lie. It is dishonest for us to suggest that it is where development discussion happens, and presents our development community as - at best, dead or incompetent, at worst deceptive and secretive. If we can't even keep a link to a development forum working for the people the link says it is for, then that reflects *very* poorly on us.

Beyond that, this is a change that can be reverted or changed in minutes. If the situation with google groups is ever fixed, we can change it here almost instantly. In fact, having it listed as archival for now and then reposted to prominence later when the google groups is fixed might actually **help** drive people to the groups, because it is a fresh change. Unlike right now, where people just try to go to it, fail, and then forever have it blacklisted in their memory as a deadlink to just ignore. It takes a lot more effort to get people to go back and try a deadlink again than it does to get people to try something *new*

And as a further benefit: Having the google group link moved to an "Archival" link section with an explanation might help MZ realize just how much it needs to be fixed. I'm sure it has been mentioned to him in an email, but nothing's been fixed yet. And to be honest, it could even be fixed now, and I'd never know, because I'm one of the lucky few who visited it before and can still reach it, so nothing's changed. And I'm not going to bother logging out to login to a fresh account just to test this on a regular basis. So having it moved to an Archival section provides the community with a visual indicator for whenever the groups are actually fixed: When the link gets restored to a prominent position, we know it is supposed to be fixed again.
